### PR TITLE
Pull Request for Icon Component Demos

### PR DESCRIPTION
**Pull request by bbrennanbasis**
- Added two demos using icons in a TabbedPane class
 - Added a folder in resources called icon to hold five custom icons

custom-pool-icons

### CODE REVIEW

**Comment by bbrennanbasis** [comment]
@MatthewHawkins please review the demos I have currently and let me know what other demos need to be made for the Icon component.

**Comment by MatthewHawkins** [comment]
@laurenic0l Which of these files do you have slated for use in the icon docs?

**Review by MatthewHawkins — DISMISSED** [review]
These look good from a code/demo standpoint. 

@hyyan Do these fit more with what you're looking for? If so, which ones would you prefer to see on the doc site?

File: src/main/java/com/webforj/samples/views/icon/IconVariationsView.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import com.webforj.component.icons.Icon;
+import com.webforj.component.icons.TablerIcon;
+import com.webforj.component.layout.flexlayout.FlexDirection;
+import com.webforj.component.layout.flexlayout.FlexLayout;
+import com.webforj.router.annotation.FrameTitle;
+import com.webforj.router.annotation.Route;
+
+@Route
+@FrameTitle("Icon Variations")
+public class IconVariationsView extends Composite<FlexLayout> {
```
Comment (hyyan) [inline src/main/java/com/webforj/samples/views/icon/IconVariationsView.java:15]: @laurenic0l this one is good, please include it please

File: src/main/java/com/webforj/samples/views/icon/IconPoolsView.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import com.webforj.component.icons.Icon;
+import com.webforj.component.icons.TablerIcon;
+import com.webforj.component.layout.flexlayout.FlexDirection;
+import com.webforj.component.layout.flexlayout.FlexLayout;
+import com.webforj.router.annotation.FrameTitle;
+import com.webforj.router.annotation.Route;
+
+@Route
+@FrameTitle("Icon Pools")
+public class IconPoolsView extends Composite<FlexLayout> {
```
Comment (hyyan) [inline src/main/java/com/webforj/samples/views/icon/IconPoolsView.java:15]: @laurenic0l remove please

File: src/main/java/com/webforj/samples/views/icon/IconMenuView.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+import com.webforj.component.tabbedpane.TabbedPane;
+import com.webforj.component.tabbedpane.Tab;
+import com.webforj.component.Composite;
+import com.webforj.component.Theme;
+import com.webforj.component.html.elements.Div;
+import com.webforj.component.tabbedpane.TabbedPane.Alignment;
+
+@Route
+@FrameTitle("Icon Menu")
+public class IconMenuView extends Composite<Div> {
```
Comment (hyyan) [inline src/main/java/com/webforj/samples/views/icon/IconMenuView.java:20]: @laurenic0l remove please

File: src/main/java/com/webforj/samples/views/icon/IconItemCardView.java [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+import com.webforj.component.layout.flexlayout.FlexDirection;
+import com.webforj.component.layout.flexlayout.FlexJustifyContent;
+import com.webforj.component.layout.flexlayout.FlexLayout;
+import com.webforj.router.annotation.FrameTitle;
+import com.webforj.router.annotation.Route;
+
+@Route
+@FrameTitle("Icon Item Cards")
+@InlineStyleSheet("context://css/icon/iconItemCard.css")
+public class IconItemCardView extends Composite<FlexLayout> {
```
Comment (hyyan) [inline src/main/java/com/webforj/samples/views/icon/IconItemCardView.java:25]: @laurenic0l remove please

File: src/main/java/com/webforj/samples/views/icon/IconBasicsView.java [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+import com.webforj.component.icons.Icon;
+import com.webforj.component.icons.TablerIcon;
+import com.webforj.component.layout.flexlayout.FlexDirection;
+import com.webforj.component.layout.flexlayout.FlexLayout;
+import com.webforj.router.annotation.FrameTitle;
+import com.webforj.router.annotation.Route;
+
+@Route
+@FrameTitle("Icon Basics")
+public class IconBasicsView extends Composite<FlexLayout> {
```
Comment (hyyan) [inline src/main/java/com/webforj/samples/views/icon/IconBasicsView.java:13]: @laurenic0l this one is good, please include it but format the code first

File: src/main/java/com/webforj/samples/views/icon/IconActionButtonsView.java [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+import com.webforj.component.icons.Icon;
+import com.webforj.component.icons.TablerIcon;
+import com.webforj.component.layout.flexlayout.FlexDirection;
+import com.webforj.component.layout.flexlayout.FlexLayout;
+import com.webforj.router.annotation.FrameTitle;
+import com.webforj.router.annotation.Route;
+
+@Route
+@FrameTitle("Icon Action Buttons")
+public class IconActionButtonsView extends Composite<FlexLayout> {
```
Comment (hyyan) [inline src/main/java/com/webforj/samples/views/icon/IconActionButtonsView.java:17]: @laurenic0l this one is good, please include it but format the code first

**Review by hyyan — CHANGES_REQUESTED** [review]

File: src/main/java/com/webforj/samples/views/icon/IconPrefixSuffixView.java [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+@FrameTitle("Icon Action Buttons")
+public class IconPrefixSuffixView extends Composite<FlexLayout> {
+  
+  public IconPrefixSuffixView() {
+    getBoundComponent().setDirection(FlexDirection.ROW).setMargin("var(--dwc-space-l)")
+    .setSpacing("var(--dwc-space-l)");
+    
+    Icon arrow = TablerIcon.create("arrow-narrow-right");
+    Button nextButton = new Button("Next").setSuffixComponent(arrow)
+    .setTheme(ButtonTheme.SUCCESS);
```
Comment (hyyan) [inline src/main/java/com/webforj/samples/views/icon/IconPrefixSuffixView.java:23]: change theme to primary please

File: src/main/resources/css/icon/iconItemCard.css [hunk 8]
```diff
@@ -0,0 +1,38 @@
+.item-card {
```
[the remark this answers is not in the document]
Comment (laurenic0l) [reply src/main/resources/css/icon/iconItemCard.css]: removed

**Pull request merged**